.github/workflows/test.yml

```diff
@@ -51,4 +55,33 @@ jobs:
         export LD_LIBRARY_PATH=`python -m awkward.config --libdir`:$LD_LIBRARY_PATH
         python config/FCCAnalysisRun.py examples/FCCee/higgs/mH-recoil/mumu/analysis_stage1.py --output myoutput.root --files-list root://eospublic.cern.ch//eos/experiment/fcc/ee/generation/DelphesEvents/spring2021/IDEA/p8_ee_Zbb_ecm91_EvtGen_Bc2TauNuTAUHADNU/events_131527278.root
         '
-
+    - name: Run benchmark
+      run: |
+        docker exec CI_container /bin/bash -c 'yum install -y jq
+        cd ./Package
+        source ${{ matrix.SETUP }}
+        export LOCAL_DIR=$PWD
+        export PYTHONPATH=$PWD:$PYTHONPATH
+        export LD_LIBRARY_PATH=$PWD/install/lib:$LD_LIBRARY_PATH
+        export ROOT_INCLUDE_PATH=$PWD/install/include/FCCAnalyses:$ROOT_INCLUDE_PATH
+        export LD_LIBRARY_PATH=`python -m awkward.config --libdir`:$LD_LIBRARY_PATH
+        python config/FCCAnalysisRun.py examples/FCCee/higgs/mH-recoil/mumu/analysis_stage1.py --test --bench
```

Comment:
I think you can just output the benchmarks when running this same file in CTest ( add --bench here: https://github.com/HEP-FCC/FCCAnalyses/blob/master/tests/CMakeLists.txt#L7)